common/log/log.go

```diff
@@ -85,7 +87,19 @@ func GetGID() uint64 {
 	return n
 }
 
-var Log *Logger
+var globalLogger unsafe.Pointer
```

Comment:
validate pointer must not nil after init?

Reply:
there is a init function to call InitLog, so can gurantee globalLogger will never be nil.